docs/tutorials/resource-migration.md

```diff
@@ -0,0 +1,235 @@
+3. If not, you shall add `--feature-gates=PropagationPolicyPreemption=true` into that field.
+
+### Preset resource in a member cluster
+
+To simulate resources already exist in the member cluster, we deploy some simple Deployments and Services to `member1` cluster.
+
+**Step 3: Write the code**
+
+Create new file `/tmp/deployments-and-services.yaml` and copy text below to it:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deploy
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:latest
+        ports:
+        - containerPort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx-svc
+spec:
+  selector:
+    app: nginx
+  type: NodePort
+  ports:
+  - port: 80
+    nodePort: 30000
+    targetPort: 80
+```
+
+**Step 4: Run the command**
+
+```shell
+$ kubectl --context member1 apply -f /tmp/deployments-and-services.yaml
+deployment.apps/nginx-deploy created
+service/nginx-svc created
+deployment.apps/hello-deploy created
+service/hello-svc created
+```
+
+Thus, we can use `member1` as the cluster with existing resources, while `member2` as a bare cluster.
+
+## Tutorials
+
+### Migrate all the resources to Karmada
+
+1）Apply the local file `/tmp/deployments-and-services.yaml` written just now to Karmada control plane too.
+
+**Step 5: Run the command**
+
+```shell
+➜ ✗ kubectl --context karmada-apiserver apply -f /tmp/deployments-and-services.yaml
+deployment.apps/nginx-deploy created
+service/nginx-svc created
+deployment.apps/hello-deploy created
+service/hello-svc created
+```
+
+2）Write a `PropagationPolicy` to the local file `/tmp/pp-for-migrating-deployments-and-services.yaml`.
+
+**Step 6: Write the code**
+
+Create new file `/tmp/pp-for-migrating-deployments-and-services.yaml` and copy text below to it:
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: migrate-pp
+spec:
+  conflictResolution: Overwrite
+  placement:
+    clusterAffinity:
+      clusterNames:
+      - member1
+  priority: 0
+  resourceSelectors:
+  - apiVersion: apps/v1
+    kind: Deployment
+  - apiVersion: v1
+    kind: Service
+  schedulerName: default-scheduler
+```
+
+**Note:**
+
+You should pay attention to two fields： 
+
+* `spec.conflictResolution: Overwrite`：the value must be [Overwrite](https://github.com/karmada-io/karmada/blob/master/docs/proposals/migration/design-of-seamless-cluster-migration-scheme.md#proposal).
+* `spec.resourceSelectors`：selecting resources to migrate, you can define your custom [ResourceSelector](https://karmada.io/docs/userguide/scheduling/override-policy/#resource-selector).
+
+3）Apply `/tmp/pp-for-migrating-deployments-and-services.yaml` to Karmada control plane. 
```

Comment:
The index `1), 2)...` and `Step 1, Step 2 ...` might be confusing.

Reply:
yes, realized it~